src/sentry/static/sentry/app/actionCreators/navigation.tsx

```diff
@@ -34,6 +35,6 @@ export function navigateTo(to, router) {
   }
 }
 
-export function setLastRoute(route) {
+export function setLastRoute(route: string) {
```

Comment:
maybe it's good to have an `Enum` with our routes that we really have 😉

Reply:
@priscilawebdev How does that work with routes that have params in them? Either way, I fear creating such an enum would be an enormous amount of work.

Reply:
I am not so familiar with our routes, I would need to do some research to answer you, but if it is too much work, I believe you can leave it as it is now